### ISSUE TITLE
automatically search for pagelinks

### ISSUE THREAD

**Issue by hajika** [claudioc/jingo#217]
after typing `[[` jingo could search for suggestions of existing wikipages to easily link them.

**Comment by claudioc** [comment]
Good idea. The current "client side" jingo (the js that runs in the browser) it's very poor and it will be greately improved in the next major
